docs/.ruby-version

```diff
@@ -0,0 +1 @@
+3.0.1
```

Comment:
Looks like this file did not exist before - do we need this to be checked in?

Reply:
Probably not? I'll remove it.